src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantsController.kt

```diff
@@ -52,8 +55,12 @@ class ParticipantsController(
                 cohortId = payload.cohortId,
                 name = payload.name,
                 projectIds = payload.projectIds?.toSet() ?: emptySet()))
+    val details =
+        projectAcceleratorDetailsService.fetchParticipantProjectDetails().associateBy {
```

Comment:
It seems a bit wasteful to me to fetch the details for every project in every participant on the system, and then discard all but the handful of them that are part of the newly-created participant. Can we instead add a fetch function to get the details for the projects in a single participant?